### PR TITLE
OperationGuide

### LINKED ISSUE #2: How to use SQLFlow in Zeppelin
**The use  are divided into the following steps:**

1. Enter the Zeppelin home page, click the user button in the upper right corner, and select “**interpreter**” to enter the parameter configuration page.
interpreter

2. Fill in 5 configuration parameters, namely:
- **sqlflow.serverAddr**: sqlflow server address
- **mysql.username**: database login user name
- **mysql.password**: database login password
- **mysql.serverAddr**: database server address
- **mysql.databaseName**: database name
config2

3. Go back to home page and create a new note. Select `sqlflow` as the default interpreter, which is shown in the figure below:
create

   Create_New_Note

4. Start to use sqlflow syntax in the open notebook, and then run to perform a functional test.
As the case shown in the figure below, the `DNNClassifier` classifier is trained on the [iris](https://en.wikipedia.org/wiki/Iris_flower_data_set) data set, and the trained `DNNClassifier` is used to predict three types of iris. The drawing can more intuitively show the incidence of each feature field on the model prediction more intuitively.

    Note: Each paragraph must begin with "**%sqlflow**".
    twoparagraph

### PR DESCRIPTION
**Pull request by Klay1124**
fix #2

How to use SQLFlow in Zeppelin.

### CODE REVIEW

File: OperationGuide.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿**The use  are divided into the following steps:**
```
Comment (weiguoz) [inline OperationGuide.md]: Use `#` to identify a topic

Comment (weiguoz) [reply OperationGuide.md]: How about _moving_ the documents into a single directory `doc`?

File: OperationGuide.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- **sqlflow.serverAddr**: sqlflow server address
+- **mysql.username**: database login user name
+- **mysql.password**: database login password
+- **mysql.serverAddr**: database server address
+- **mysql.databaseName**: database name
+   ![参数配置页面2](https://user-images.githubusercontent.com/65579930/119923915-3aaf2d00-bfa5-11eb-92ff-78f403ad555f.jpg)
+3. Go back to home page and create a new note. Select `sqlflow` as the default interpreter, which is shown in the figure below:
+![create](https://user-images.githubusercontent.com/65579930/112590118-419eaf80-8e3d-11eb-80a4-9611ffde8115.jpg)
+
+   ![Create_New_Note](https://user-images.githubusercontent.com/65579930/112590123-4499a000-8e3d-11eb-96ff-a30868244ce8.jpg)
```
Comment (weiguoz) [inline OperationGuide.md]: Remove the 2 spaces.

File: OperationGuide.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+![interpreter](https://user-images.githubusercontent.com/65579930/112589464-18c9ea80-8e3c-11eb-9b61-b39d5ceb731a.jpg)
+
+2. Fill in 5 configuration parameters, namely:
+- **sqlflow.serverAddr**: sqlflow server address
+- **mysql.username**: database login user name
+- **mysql.password**: database login password
+- **mysql.serverAddr**: database server address
+- **mysql.databaseName**: database name
+   ![参数配置页面2](https://user-images.githubusercontent.com/65579930/119923915-3aaf2d00-bfa5-11eb-92ff-78f403ad555f.jpg)
+3. Go back to home page and create a new note. Select `sqlflow` as the default interpreter, which is shown in the figure below:
```
Comment (weiguoz) [inline OperationGuide.md]: There is a blank line between item `1` and `2`, but no blank line between `2` and `3`

**Review by weiguoz — CHANGES_REQUESTED** [review]

File: OperationGuide.md (hunk 1 above)
Comment (Klay1124) [reply OperationGuide.md]: OK. A new branch has been pulled to submit this code. This branch will close.

File: OperationGuide.md (hunk 2 above)
Comment (Klay1124) [reply OperationGuide.md]: useless space removed.

File: OperationGuide.md (hunk 3 above)
Comment (Klay1124) [reply OperationGuide.md]: The format has been changed to be consistent.

**Review by weiguoz — DISMISSED** [review]
LGTM

**Review by weiguoz — APPROVED** [review]

**Pull request merged**